board/piksiv3/rootfs-overlay/etc/init.d/template_runsv.inc.sh

```diff
@@ -0,0 +1,94 @@
+# /etc/init.d template for processes
+
+# name=""
+# cmd=""
+# dir="/"
+# user=""
+# source template.inc.sh
+
+source /etc/init.d/common.sh
+
+stdout_log="/var/log/$name.log"
+stderr_log="/var/log/$name.err"
+
+tag=tmpl_runsv
+fac=daemon
+
+pid_file="/var/service/${name}/supervise/pid"
+
+_setup_svdir()
+{
+  mkdir -p /etc/sv/${name}/control
+
+  if [[ -z "$user" ]]; then
+    echo "Error: the 'user' variable must not be empty"
+    exit 1
+  fi
+
+  echo "#!/bin/ash"                                   > /etc/sv/${name}/run
+  echo "cd ${dir}"                                   >> /etc/sv/${name}/run
+  echo "echo Starting ${name}... \\"                 >> /etc/sv/${name}/run
+  echo "  | logger -t ${tag} -p ${fac}.info"         >> /etc/sv/${name}/run
+  echo "exec chpst -u $user:$user $cmd \\"           >> /etc/sv/${name}/run
+  echo "  1>>${stdout_log} \\"                       >> /etc/sv/${name}/run
+  echo "  2>>${stderr_log}"                          >> /etc/sv/${name}/run
+
+  chmod +x /etc/sv/${name}/run
+
+  echo "#!/bin/ash"                                   > /etc/sv/${name}/finish
+  echo "echo Service ${name} exited... \\"           >> /etc/sv/${name}/finish
+  echo "  | logger -t ${tag} -p ${fac}.info"         >> /etc/sv/${name}/finish
+  echo "sleep 1"                                     >> /etc/sv/${name}/finish
+
+  chmod +x /etc/sv/${name}/finish
+
+  ln -sf /etc/sv/${name} /var/service/${name}
+}
+
+sv_is_running()
+{
+  local name=$1; shift
+  local svc_dir=/var/service/${name}
+
+  sv status $svc_dir | grep -q "^run"
+}
+
+do_start()
+{
+  _setup_permissions
+  _setup_svdir
+
+  for i in `seq 1 10`; do
+    if sv_is_running $name; then
+      break
+    fi
+    sleep 0.1;
+  done
+
+  if ! sv_is_running $name; then
+    echo "Error: service '${name}' failed to start after 1 second..."
+  fi
+}
+
+case "$1" in
+  start)
+    do_start
+    ;;
+  stop)
+    sv down /var/service/${name}
```

Comment:
RE: [this](https://github.com/swift-nav/piksi_buildroot/pull/595#discussion_r185118986) comment. Since we are using down here maybe we need to have a stateful `start` that can use `up` instead if not first init